data/governments.txt

```diff
@@ -677,6 +678,77 @@ government "Independent (Killable)"
 	"friendly hail" "friendly civilian"
 	"hostile hail" "hostile civilian"
 
+# These are Korath- and pirate-friendly humans that dislike the Syndicate
+government "Independent (Korath-Allied)"
+	swizzle 9
+	color "governments: Independent"
+	"display name" "Independent"
+
+	"player reputation" 10
+	"bribe" .05
+	"fine" 0
+	"friendly hail" "friendly civilian"
+	"hostile hail" "hostile civilian"
+	"attitude toward"
+		"Korath (Civilian)" .01
+
+	"friendly hail" "friendly civilian"
+	"hostile hail" "hostile civilian"
+
+government "Independent (Korath-Allied) Port"
+	swizzle 9
+	color "governments: Independent"
+	"display name" "Independent"
+
+	"player reputation" 10
+	"bribe" .05
+	"fine" 0
+	"friendly hail" "friendly civilian"
+	"hostile hail" "hostile civilian"
+	"attitude toward"
+		"Syndicate" -.01
+		"Syndicate (Extremist)" -.01
+		"Republic" -.01
+		"Republic that won't enter wormhole" -.01
+		"Navy Intelligence" -.01
+		"Republic Intelligence" -.01
+		"Independent (Korath-Allied)" 1
+		# Negative for Korath ensures the Korath raiding groups will not come
+		# from Independent (Korath-Allied) Port planets and stations.
+		"Korath" -.01
+		"Kor Alaji" -.01
+		"Kor Isaha" -.01
+
+# Used for smuggling sites and other restricted-access stations. Allows Korath to land.
+government "Independent (Korath-Allied) Restricted"
+	swizzle 9
+	color "governments: Independent"
+	"display name" "Independent"
+
+	"player reputation" 10
+	"bribe" .05
+	"fine" 0
+	"friendly hail" "friendly civilian"
+	"hostile hail" "hostile civilian"
+	"attitude toward"
+		"Syndicate" -.01
+		"Syndicate (Extremist)" -.01
+		"Merchant" -.01
+		"Pirate" -.01
+		"Republic" -.01
+		"Republic that won't enter wormhole" -.01
+		"Hai" -.01
+		"Hai (Wormhole Access)" -.01
+		"Remnant" -.01
+		"Quarg" -.01
+		"Navy (Oathkeeper)" -.01
+		"Free Worlds" -.01
+		"Wanderer" -.01
```

Comment:
```suggestion
```
Wanderer escorts cannot get outside wanderer/regular korath space

Reply:
The Deep River equipped with a Jump Drive that you bring carrying tribute to the Unfettered can.